debian/control

```diff
@@ -3,25 +3,14 @@ Section: python
 Priority: optional
 Maintainer: Farsight Security, Inc. <software@farsightsecurity.com>
 Build-Depends:
- cython (>= 0.15~),
```

Comment:
building on debian stretch fails:
```
$ python3 setup.py build 
Traceback (most recent call last):
  File "setup.py", line 15, in <module>
    from Cython.Distutils import build_ext
ImportError: No module named 'Cython'
```
our setup.py starts with:
`from Cython.Distutils import build_ext`

Reply:
Makes sense, you would need that; cython3 installed in the line below for those building via Debian.